src/cdk-experimental/selection/row-selection.ts

```diff
@@ -26,7 +26,8 @@ import {CdkSelection} from './selection';
   },
 })
 export class CdkRowSelection<T> {
-  @Input('cdkRowSelectionValue') value: T;
+  // We need an initializer here to avoid a TS error.
+  @Input('cdkRowSelectionValue') value: T = undefined as unknown as T;
```

Comment:
You should be able to reduce these `as unknown as T` casts to `undefined!`.

Reply:
Updated.